lang/lang/editor/Syntax.esv

```diff
@@ -14,51 +14,14 @@ language
   block comment : "/*" * "*/"
   fences        : [ ] ( ) { }
 
-colorer
-
-  keyword  = 153 51 153
-  number   = 17 131 22
-  string   = 177 47 2
-  varid    = 0 0 128
-  classid  = 0 0 255
-  funcid   = 153 51 0
-  operator = 0 0 0
-  comment  = 63 127 95 italic
-
-colorer
-
-  keyword     : keyword
-  identifier  : varid
-  string      : string
-  number      : number
-  var         : varid
-  operator    : operator
-  layout      : comment
-
-  ID          : varid
```

Comment:
Why are these coloring definitions removed? Differentiating between the different kinds of identifiers with colors was quite nice.

Reply:
I'm pretty sure that they are still colored, they are now generated instead by sdf3 instead of specified here. I'll verify tomorrow

Reply:
They were not colored, changed it back. Also expanded `ID` to the separate semantic sorts